services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -645,6 +645,25 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  update_env_vars_from_varchar_to_text()
+
+  BEGIN
+    IF NOT EXISTS (
```

Comment:
this `if not exist` means that it will check if it exist and if not it will runt he `ALTER TABLE`.
In your case though it will always already exist, therefore the alter table code will never run :)

see https://github.com/amazeeio/lagoon/pull/1133/files#diff-b44d678a275f87787a48186fffc33465R584 for an example on how to change an existing field

Reply:
Resolved in https://github.com/amazeeio/lagoon/pull/1133/commits/afcc2c61da5335dde00ff8e9ecf55932dbe03e74 .